sdk/eventhub/Azure.Messaging.EventHubs/tests/Diagnostics/DiagnosticsTests.cs

```diff
@@ -34,6 +34,8 @@ namespace Azure.Messaging.EventHubs.Tests
     /// </remarks>
     ///
     [NonParallelizable]
+    [TestFixture]
+    [Ignore("Tests intermittently failing in CI; Investigation needed")]
```

Comment:
Tracking issue?

Reply:
I'm actively working on this now;  if I don't have a PR out with a fix by mid-day, I'll open one to track so that it doesn't get lost.